redisinsight/ui/src/utils/formatters/java-date.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectInputStream, JavaSerializable } from 'java-object-serialization'
+
+export default class JavaDate implements JavaSerializable {
+  time: bigint = 0n
+
+  readObject(stream: ObjectInputStream): void {
+    this.time = stream.readLong()
+  }
+
+  readResolve(): any {
```

Comment:
could you please add type here

Reply:
I have removed the type to let the type inference kick-in.